teamcity/PublishCSPForUnity.py

```diff
@@ -163,7 +162,7 @@ def main():
         f.write(minimal_meta_template.render(guid=generate_guid(), importer_type="Default"))
 
     for path in args.csharp_artifact_paths:
-        dir_util.copy_tree(path, source_output_directory)
+        shutil.copytree(path, source_output_directory)
```

Comment:
`shutil.copytree` is notorious for failing if the target directories already exist. Since Python 3.8 though, this can be accounted for by passing a `dirs_exit_ok` parameter. More info [here](https://stackoverflow.com/a/64609884).

I expect we want to change this line to be:
`shutil.copytree(path, source_output_directory, dirs_exist_ok=True)`

Reply:
(I would expect our scripts to fail without making this change).

Reply:
Agreed, change made.

Reply:
Looks good, thanks!